### ISSUE TITLE
Adding new sensors only download the last 24h

### ISSUE THREAD

**Issue by carlosparadis** [erdl/legacy-scrape-util#11]
This behavior was observed today when adding the egauges for a new house (see below the list of egauges noted by Eileen). The lines that are doing this are noted below:

https://github.com/erdl/legacy-scrape-util/blob/2c4a718e3697cfef84493cdb1245baeb0d3b4693/src/acquire/egauge.py#L60
https://github.com/erdl/legacy-scrape-util/blob/2c4a718e3697cfef84493cdb1245baeb0d3b4693/src/acquire/egauge.py#L61

Additionally, line 61 seems to have a step of 365 days, assuming the unix timestamp is in seconds, whereas line 60 is a single day. @jygh98 could you re-explain how this 1 year step works?

```
'e785' = 785, 'e786' = 786, 'e795' = 795, 'e792' = 792

'e785-Whole-house' = 187
'e785-Water-heater' = 188
e785-ACC = 189
e785-AHU = 190
e785-Dryer = 191
e785-Range = 192
'e785-Kitchen-refrigerator' = 193
e785-Other = 194

e786-PV	= 195

'e795-Whole-house' = 196
'e795-Water-heater' = 197
e795-ACC = 198
e795-AHU = 199
e795-Dryer = 200
e795-Range = 201
'e795-Kitchen-refrigerator' = 202
e795-Other = 203

e792-PV	= 204
``` 

This is an undesirable behavior. One would expect that the data pulled would include data from the day of the deployment up to the most recent on the first run, and then keeping updating. I am not positive we will fix this, as we are moving to the Notebooks, but I wanted to at least have this problem listed here. 

Was there a reason why this is done? Maybe a quota on data size request or inability for the script to figure out a starting date? 

p.s.: Were the respective egauges also added to the `dhhl` database on `sensor_info` table?

**Comment by jygh98** [comment]
So for the first part, I noticed a very interesting contradiction from forrest's egauge walkthrough video.  If you start at time [34:15](https://www.youtube.com/watch?v=IcQST8rVRtE&feature=youtu.be) he talks about how init and start time works.  But closer to 34:40 when Eileen asks what are those default times, he says that default start time is 2 weeks.  Which doesn't make sense as by 31536000 seconds it is indeed 1 year.  And the dhhl config.toml file doesn't have a start or end time stated.  So I am not too sure whether that step time is actually 2 weeks like Forrest says it is, or if it is 1 year, we can try test this out.

As for why there were no functionality to start pulling data from when the egauge was first installed, I am not too sure.  Possibly I think he would never have to worry about having to find a start date, as I think he would normally already have the new egauges added to the config file beforehand.

**Comment by carlosparadis** [comment]
> But as the video continues to 34:40 when Eileen asks what are those default times, he says that default start time is 2 weeks. But this doesn't make sense as 31536000 seconds is 1 year. And the dhhl config.toml file doesn't have a start or end time stated. So I am not too sure how Forrest converts that step time to be actually 2 weeks like he said it is or if it is 1 year.

I remember this (not the video, but the place in code). And yes, I too have no idea on why this. However, I guess it is not worth guessing this one since we already observed what the code timestamp says matches what happened (?). 


> As for why there were no functionality to start pulling data from when the egauge was first installed, I am not too sure. Possibly I think he did not have to worry about having to find a start date as I think he would normally already have the new egauges added to the config file beforehand.

Yes, this is the case. I confirmed it today with Eileen. They never had to add it later as we are doing now, which never prompted to the situation of initializing scrape-util with previous data. I can only imagine Forrest just did a quick hack with `requests` library to get any missing data and send to Eileen. 

TL;DR It is not a **bug**, or a wrong **setting**, but a **missing feature**!. Since we are hoping to move away from it, I would rather not invest time in figuring out how to do that, and just use the notebooks as-is after I add a dependency to save the data as a reading table format so you don't have to reshape. 

I am closing this issue and marking it as a `wont-fix` since it would involve adding such feature to scrape-util. You already have a workaround with the 12 python notebook sitting by until I finish the notebook next week. Feel free to re-open if you feel I am missing anything.

p.s.: The youtube video is unlisted, and linking it here may trigger crawlers to index on Google, which defeats the purpose, so I edited-out. In the future, it is best you refer to the Google Sites page and say where it is via its name because Google Sites will require authentication to see anything.
